exporter/dorisexporter/README.md

```diff
@@ -24,13 +24,12 @@ The following configuration options are supported:
   * `logs` (default = otel_logs) The table name for logs.
   * `traces` (default = otel_traces) The table name for traces.
   * `metrics` (default = otel_metrics) The table name for metrics.
-* `create_schema` (default = true) Whether databases and tables are created automatically.
-* `mysql_endpoint` The mysql protocol address to create the schema; ignored if `create_schema` is false.
+* `create_schema` (default = true) Whether databases and tables are created automatically in doris.
+* `mysql_endpoint` The mysql protocol address of doris. Only use to create the schema; ignored if `create_schema` is false.
```

Comment:
for later - do you think you'll want to update the schema down the road?

Reply:
Yes. This schema corresponds to the schema of opentelemetry.